servers/dart_server.dart

```diff
@@ -6,16 +6,19 @@ const String _HOST = '0.0.0.0';
 const String _GREET = 'Hello World';
 
 _startServer(arg) async {
-  var server = await HttpServer.bind(_HOST, 9292, shared: true);
-  await for (HttpRequest request in server) {
-    request.response
+  print('Start server');
+  final server = await HttpServer.bind(_HOST, 9292, shared: true);
+  server.listen((HttpRequest request) {
+      request.response
       ..write(_GREET)
       ..close();
-  }
+  });
 }
 
-void main() {
-  for (int i = 0; i < Platform.numberOfProcessors; i++)
+void main(List<String> args) {
+  final threads = (args.length>0?int.parse(args[0]):Platform.numberOfProcessors);
+  print('CPU: ${Platform.numberOfProcessors}, Threads: ${threads}');
+  for (int i = 1; i < threads; i++)
     Isolate.spawn(_startServer, null);
   _startServer(null);
```

Comment:
@costajob because the main thread also start the web server therefore 'i = 1'

Reply:
Creating threads is not like spawning processes. Threads are cheaper (indeed you can have hundreds on a Web server), processes are not (typically you will have one process per hyper-thread). You can check by your own throughput and memory consumption by eliminating one thread: nothing change